studies/migrations/0036_translate_old_registrations.py

```diff
@@ -0,0 +1,169 @@
+# Generated by Django 4.2.17 on 2025-03-26 13:09
+
+from django.db import migrations
+
+# This custom data_migration attempts to translate the old registration data,
+# namely:
+# - Observation.registratons etc.
+# - Intervention.measurement
+# - Task.registrations etc.
+# to the new system. This system is based on how we asked about registrations
+# for tasks.
+
+# this dictionary translates observation registration pk's to the new registration pk's
+# The keys are the pk's of observation registrations and the values of the corresponding
+# new style registrations.
+
+observation_registration_translation = {
+    # logdata
+    1: 8,
+    # audio-recording
+    2: 1,
+    # video-recording
+    3: 2,
+    # notes
+    4: 9,
+    # other
+    5: 10,
+}
```

Comment:
Using the object pk in this dict is a bit scary, because we don't know for sure what's in the database.

I think that in this case it would be better to match based on description string. It's self-documenting, and on the FROM-side it provides an assertion that we're working with the right data. We can just fail if a description is not found in the table.

On the TO-side using pk's is fine though, we just defined those so we can be pretty sure about them.